stl/src/memory_resource.cpp

```diff
@@ -8,17 +8,18 @@
 _STD_BEGIN
 namespace pmr {
 
-    static memory_resource* _Default_resource = nullptr;
+    static memory_resource* _Default_resource{nullptr};
 
-    extern "C" _CRT_SATELLITE_1 _Aligned_new_delete_resource_impl* __cdecl _Aligned_new_delete_resource() noexcept {
+    _EXTERN_C
+    _CRT_SATELLITE_1 _Aligned_new_delete_resource_impl* __cdecl _Aligned_new_delete_resource() noexcept {

```

Comment:
The `extern "C"` changes in this file are unrelated to the tag changes in the rest of the PR. While this transformation doesn't change the meaning of the code, it introduces inconsistency with the declarations in `<memory_resource>`. If you believe that we should consistently rely on `_EXTERN_C` blocks for declarations and definitions, please file an enhancement issue so we can make a decision - and then if we decide on a convention, it can be cleaned up in a separate PR.

The general principle is that large code transformations/cleanups should be performed in separate PRs for ease of code reviewing and history reading. Mixing in very small changes can be OK (e.g. comment changes, or the ` _Default_resource{nullptr}` initialization change above - that one is at least thematically related to "brace-initialize more things").

I noticed similar changes in #487 but didn't object then because they were small and increased consistency between the declarations and definitions. I probably should have mentioned my concerns though, apologies for not doing so earlier.

Reply:
Yeah totally right !, let's revert and consider another PR for these changes

> The `extern "C"` changes in this file are unrelated to the tag changes in the rest of the PR. While this transformation doesn't change the meaning of the code, it introduces inconsistency with the declarations in `<memory_resource>`. If you believe that we should consistently rely on `_EXTERN_C` blocks for declarations and definitions, please file an enhancement issue so we can make a decision - and then if we decide on a convention, it can be cleaned up in a separate PR.

Reply:
you don't need to apologies 😉, what do you think about a file  an enhancement issue and assign it to me ? 
> I noticed similar changes in #487 but didn't object then because they were small and increased consistency between the declarations and definitions. I probably should have mentioned my concerns though, apologies for not doing so earlier.